wiki/Wiki.md

```diff
@@ -0,0 +1,80 @@
+## Table of contents
+
+* [Contributing to the wiki](#contributing-to-the-wiki)
+   * [Opening a pull request](#opening-a-pull-request)
+   * [Failed to push changes to wiki upon PR merge.](#failed-to-push-changes-to-wiki-upon-pr-merge)
+* [Implementation details](#implementation-details)
+   * [Wiki architecture](#wiki-architecture)
+   * [When you make changes through the web interface](#when-you-make-changes-through-the-web-interface)
+   
+
+## Contributing to the wiki
+
+If you notice something about the wiki that could be improved, please let us know! There are a couple of ways you can do so:
+
+* If the improvement you have in mind is big, you want feedback before you start working on it, or you don't have time to make the change yourself, [open an issue](https://github.com/oppia/oppia-web-developer-docs/issues/new/choose) in the source repository.
+* If you can make the change yourself, see the instructions below for opening a pull request (PR).
+
+### Opening a pull request
+
+To contribute to the wiki, you should follow these steps:
+
+1. Checkout the `develop` branch and pull in the latest changes from upstream:
+
+   ```console
+   git checkout develop
+   git pull upstream develop
+   ```
+
+2. Create a new branch for your changes:
+
+   ```console
+   git checkout -b {{branch name}}
+   ```
+
+3. Make your changes, being sure to follow our [style guide](https://github.com/oppia/oppia/wiki/Wiki-style-guide). You can use whatever text editor you prefer for this.
```

Comment:
I think we need to add `style guide` as per oppia-android.
But for now It's good to have oppia-web.
What your suggestions?

Reply:
We aim to maintain the same style guide across both repos.
Can you elucidate how the oppia-android style guide may differ from that of oppia-web?

Reply:
I think the screenshots guide can be used by both repos. Maybe if we ever see a user-case that requires us to create a separate style guide then we can make one for Oppia-android. What do you think?

Reply:
Yes that's looks good to me, we can mame separate guide. If something more specific we need. For now it's good to have oppia-web. Thanks.

Reply:
I think having something like a style guide makes sense, but I'm also not sure how exactly the rules/patterns are determined. This might become more clear (to me at least) with more PRs in the future changing the wiki (since, at the moment, most of the wiki pages have been created/edited by a small number of individuals so consistency hasn't been as much of an issue).